packages/containerd/containerd-config-toml_k8s_nvidia_containerd_sock

```diff
@@ -21,6 +21,7 @@ address = "/run/containerd/containerd.sock"
 [plugins."io.containerd.grpc.v1.cri"]
```

Comment:
Leaving this here for the lack of a better place to post it:

In your commit message, you can mention that CDI support is required so that containerd updates the OCI spec based off the CDI specifications generated by the Device Plugin.